### PR TITLE
Document new AT_LIGHT_PASS builtin (2.1)

### PR DESCRIPTION
**Pull request by RandomShaper**

### CODE REVIEW

File: learning/features/shading/shading_language.rst [hunk 1]
```diff
@@ -377,41 +377,45 @@ CanvasItem (2D) - VertexShader
 +-----------------------------------+--------------------------------------------------------------------------------------------+
 | const float *TIME*                | Time (in seconds)                                                                          |
 +-----------------------------------+--------------------------------------------------------------------------------------------+
+| const bool *AT_LIGHT_PASS*        | Whether the shader is being run for a lighting pass (happens per affecting light)          |
++-----------------------------------+--------------------------------------------------------------------------------------------+
 
 CanvasItem (2D) - FragmentShader
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-+-------------------------------------+------------------------------------------------------------------------------+
```
Comment (akien-mga) [inline learning/features/shading/shading_language.rst]: Gotta love reST for this :D

Comment (RandomShaper) [inline learning/features/shading/shading_language.rst]: Yeah... I've tried to find a shorter wording by all means to avoid that cumbersome edit (and ugly diff) but nothing made sense.

Googling a bit I've found this: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

So in theory, MarkDown support ugly formatting for tables at the source code level. Maybe that would be a win.

Comment (akien-mga) [inline learning/features/shading/shading_language.rst]: Yeah but Sphinx is designed around reST, I don't particularly want to change markup language yet another time (we already went MarkDown -> Textile -> reST). reST tables are a bit cumbersome, but the rest is pretty good for documentation IMO.

**Comment by RandomShaper** [comment]
OK then! I agree with you.
